lib/bap/bap.mli

```diff
@@ -621,12 +621,12 @@ module Std : sig
       (** Converts an 'a to a string *)
       type 'a printer = Format.formatter -> 'a -> unit
 
-      (** Interconversion between string and 'a type *)
-      type 'a converter = 'a parser * 'a printer
+      (** Interconversion between string and 'a type, along with a default *)
+      type 'a converter = 'a parser * 'a printer * 'a
```

Comment:
Usually we're trying not to use unlabeled tuples of arity greater than two, especially in the interface or on a module scope, preferring to use records instead. The latter also provides a tighter typing discipline, as `(parser,printer,default)` belongs to a vast family of `'a * 'b * 'c` triples, where `{parser;printer;default}` belongs to a specific type `'a converter` that establishes explicit constraints on its fields. For the completeness sake there is another way to establish such constraint, by defining a type (not type alias): `type 'a converter = Converter of 'a parser * 'a printer * 'a`. With this definition `Converter (parser,printer,default)` also defines a value of type converter. 

Also, originally we were trying to make our converters type compatible with Cmdliner's converters, but since we start to diverge it is better to make this type abstract, and provide constructors.